media/js/base/send-to-device.js

```diff
@@ -49,34 +50,48 @@ if (typeof Mozilla === 'undefined') {
     // static value for user country code
     SendToDevice.COUNTRY_CODE = '';
 
-    SendToDevice.prototype.geoCallback;
+    SendToDevice.prototype.geoCallback; // jshint ignore:line
 
     /**
      * Initialise the form messaging and bind events.
      */
     SendToDevice.prototype.init = function() {
         if (this.$widget.length === 1) {
-            this.checkLocation();
+            this.getLocation();
             this.bindEvents();
         }
     };
 
     /**
-     * Checks to see if the visitor is located in the US
-     * using MLS https://location.services.mozilla.com/api
+     * Gets the country code of the location of the user
+     * using bedrock's /country-code.json service
      */
-    SendToDevice.prototype.checkLocation = function() {
+    SendToDevice.prototype.getLocation = function() {
         var self = this;
-        var key = this.$widget.data('key');
 
-        // should location.services.mozilla.com be slow to load,
+        // if a dev has added ?geo=<country code> to the URL
+        // we can skip the geo lookup and act as if it worked
+        if (window.location.href.indexOf('?geo=') !== -1) {
+            var urlRe = /\?geo=([a-z]{2})/i;
+            var match = urlRe.exec(window.location.href);
+            if (match) {
+                SendToDevice.COUNTRY_CODE = match[1].toUpperCase();
+                self.updateMessaging();
+                if (typeof self.geoCallback === 'function') {
+                    self.geoCallback(SendToDevice.COUNTRY_CODE);
```

Comment:
If we're switching to upper case (which is fine, IMO), we'll need to change the scripts that are currently relying on lowercase:

- [`mobile-download-desktop.js`line 42](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/mobile-download-desktop.js#L42)
- [`whatsnew-50.js` line 46](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/whatsnew/whatsnew-50.js#L46)

Reply:
I can switch to lower. No good reason to use one or the other as long as it's consistent.